.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Feature Request
+about: Request a new feature
+title: ''
+labels: feature
+assignees: ''
+---
+
+**PLEASE DO NOT OPEN FEATURE REQUEST ISSUES ON GITHUB**
+
+**Feature requests should be opened up on our dedicated [feature request hub](https://features.jellyfin.org/) so that they can be appropriately discussed and prioritized.**
+
+However, if you are willing to contribute to the project by adding a new feature yourself, then please ensure that you first review our [documentation on contributing code](https://jellyfin.org/docs/general/contributing/development.html). Once you have reviewed the documentation feel free to come back here and open an issue here outlining your proposed approach so that it can be documented, tracked and discussed by other team members.
```

Comment:
```suggestion
However, if you are willing to contribute to the project by adding a new feature yourself, then please ensure that you first review our [documentation](https://docs.jellyfin.org/general/contributing/development.html) on contributing code. Once you have reviewed the documentation, feel free to come back here and open an issue here outlining your proposed approach so that it can be documented, tracked, and discussed by other team members.
```

Reply:
We use the subdomain for documentation links so we can potentially separate the two in the future.